Scripts/Services/Revamped Dungeons/TheExodusEncounter/Items/ExodusChest.cs

```diff
@@ -1,70 +1,203 @@
+using Server.Mobiles;
+using Server.Regions;
 using System;
 
 namespace Server.Items
 {
     public class ExodusChest : LockableContainer
     {
+        public override int LabelNumber { get { return 1073403; } } // decorative box
+
+        public static Type[] RituelItem { get { return m_RituelItem; } }
+
+        private static readonly Type[] m_RituelItem = new[]
+        {
+            typeof(ExodusSummoningRite), typeof(ExodusSacrificalDagger), typeof(RobeofRite), typeof(ExodusSummoningAlter)
+        };
+
+        private static Type[] m_ImbuingEssenceIngreds =
```

Comment:
This code is pretty similar to the one in `TreasureMapChest` and `EssenceBox` classes. I would propose to extract these definitions to the `Loot` class along with other loot constructors, so you just need to call:

```
var essence = Loot.RandomEssence();
DropItem(essence);
```

You can extract the Exodus ritual items as well in a similar way.